frontend/src/views/admin/AdminDashboardView.css

```
@@ -46,3 +224,54 @@
   padding: 2px 7px;
   border-radius: 4px;
 }
+
+/* Hover-only delete button */
+:deep(.p-datatable-tbody > tr) .delete-btn { opacity: 0; transition: opacity 0.15s; }
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)